tests/app/authentication/test_user.py

```diff
@@ -22,3 +22,8 @@ def test_get_user_ik(self):
         with self.application.test_request_context():
             user = User("1", "2")
             self.assertEqual("2", user.user_ik)
+
+    def test_no_user(self):
+        with self.application.test_request_context():
+            with self.assertRaises(ValueError):
+                User("", "")
```

Comment:
Added...
Just a question, with this `if user_id and user_ik:` is it possible to have one and not have the other, or do they both have to be there?